support-frontend/assets/components/secureTransactionIndicator/secureTransactionIndicator.tsx

```diff
@@ -1,18 +1,22 @@
 import { css } from '@emotion/react';
 import { neutral, textSansBold14 } from '@guardian/source/foundations';
+import SecurePadlock from 'components/svgs/securePadlock';
```

Comment:
Just curious if this would be in curly braces normally? Obviously if it's not flagged by prettier/lint it's all good

Reply:
Good question! I made it a default export from the component which is why there are no curly braces. This is consistent with the interface of the old jsx component which was generated by `react-svg-loader` e.g.:

```
import SecurePadlock from './securePadlock.svg';
```

Very happy to switch to a named export though if that's preferable? It looks like that's generally the approach used elsewhere. What do you reckon?

Reply:
Looks like we have a [todo](https://github.com/guardian/support-frontend/blob/a5f4a1b3c8a3c32d14bbeade9601087f1e43c2ea/support-frontend/.eslintrc.js#L9 
) to remove a lint override related to this. Probably makes sense to switch to named

Reply:
Fixed in 4431c8f